src/hooks/useCopyToClipboard.tsx

```diff
@@ -1,32 +1,34 @@
-export const useCopyToClipboard = (text : string) => {
-  return copyToClipboard(text)
-};
+import { useCallback } from 'react';
 
-const copyToClipboard = (str : string) : boolean => {
+export const useCopyToClipboard = () => {
+  const copyToClipboard = useCallback((str : string) : boolean => {
```

Comment:
Move copyToClipboard into a `useCallback` that is provided by `useCopyToClipboard` hook